### PR TITLE
mgmt, servicefabricmanagedclusters, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
- https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    @Test
+    @LiveOnly
+    public void testCreateManagedCluster() {
+        ManagedCluster managedCluster = null;
+        try {
+            String clusterName = "cluster" + randomPadding();
+            String adminUser = "user" + randomPadding();
+            String adminPassWord = "!QA2ws#ED";
```
Comment (weidongxu-microsoft) [inline sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java:77]: Please check how we use the password in other live tests. Do we append random padding as well?

The reason is for security. We do not want to have a test resource with predictable password, even if this resource only exists for a few minutes.
@XiaofeiCao thoughts?

Comment (XiaofeiCao) [reply sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java:77]: Yes, this is valid concern and CI is reporting warning for fixed password. 
Previously we did random padding for password: https://github.com/Azure/azure-sdk-for-java/blob/b0f6934ec0a04faf5f6a7ee5767a1bdd7d5d34df/sdk/postgresqlflexibleserver/azure-resourcemanager-postgresqlflexibleserver/src/test/java/com/azure/resourcemanager/postgresqlflexibleserver/PostgreSqlManagerTests.java#L94-L95

I think this is a common scenario for our test pipelines. Let me see if eng sys channel knows anything(e.g. if they could generate a password for us)..

Comment (XiaofeiCao) [reply sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java:77]: @v-hongli1 Please also help fix previous password using the random padding approach. At least we could fix CredScan...
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4217712&view=logs&j=bc38e8b8-e027-53cb-48e7-2adbd1070eca&t=d1aba2c3-5b21-511d-f336-affeabd629f9&l=361

**Review by XiaofeiCao — APPROVED** [review]
Thanks. Please also help fix previous password in live test(either in this PR, or a new one):
https://github.com/Azure/azure-sdk-for-java/blob/b0f6934ec0a04faf5f6a7ee5767a1bdd7d5d34df/sdk/mongocluster/azure-resourcemanager-mongocluster/src/test/java/com/azure/resourcemanager/mongocluster/MongoClusterManagerTests.java#L83

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java (hunk 1 above)
Comment (v-hongli1) [reply sdk/servicefabricmanagedclusters/azure-resourcemanager-servicefabricmanagedclusters/src/test/java/com/azure/resourcemanager/servicefabricmanagedclusters/ServiceFabricManagedClustersManagerTests.java]: Fixed in the new version.

**Pull request merged**